Python/tests/hypergraph_constructors.py

```
@@ -0,0 +1,212 @@
+          [1, 0, 0, 0],
+          [0, 0, 0, 0],
+          [0, 0, 0, 0]],
+         [[0, 1, 0, 0],
+          [1, 0, 0, 0],
+          [0, 0, 0, 0],
+          [0, 0, 0, 0]]]
+    )
+
+    # Validate hypergraph properties
+    assert HG.order == 3
+    assert HG.uniform == True
```

Ruff (0.8.0)

Avoid equality comparisons to `True`; use `if HG.uniform:` for truth checks

Replace with `HG.uniform`

(E712)